jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter.bats

```diff
@@ -51,3 +51,66 @@ teardown() {
     unset JAVA_INITIAL_MEM_RATIO
     unset JAVA_MAX_MEM_RATIO
 }
+
+@test "verify the env vars when is set LOG_LEVEL" {
+    rm -rf /tmp/logging.properties || true
+    cp $BATS_TEST_DIRNAME/resources/logging.properties /tmp/logging.properties
+    export LOG_LEVEL="WARN"
+    export CONFIG_DIR="/tmp"
+    PACKAGES_LOG_LEVEL_CONF=org.xyz=INFO,com.acme=DEBUG
+    export PACKAGES_LOG_LEVEL_CONF
+    configure_logger_config_file
+    [[ "${LOG_LEVEL}" == WARN ]]
+    [[ "${PACKAGES_LOG_LEVEL}" == "org.xyz=INFO\ncom.acme=DEBUG" ]]
+    echo value:${JAVA_OPTS}.
+    [[ "${JAVA_OPTS}" ==  " -Djava.util.logging.config.file=/tmp/logging.properties" ]]
+    unset LOG_LEVEL
+    unset PACKAGES_LOG_LEVEL
+    unset JAVA_OPTS
+}
+
+@test "verify the logger.properties when PACKAGES_LOG_LEVEL is unset" {
+    rm -rf /tmp/logging.properties || true
+    cp $BATS_TEST_DIRNAME/resources/logging.properties /tmp/logging.properties
+    export LOG_LEVEL=INFO
+    export CONFIG_DIR=/tmp
```

Comment:
instead export this, 

do this before the tests code:
export CONFIG_DIR=/tmp
setup() {
  cp $BATS_TEST_DIRNAME/resources/logging.properties ${CONFIG_DIR}/ogging.properties
}

do the same for the logging.properties copy and on other test cases